ci/cloudbuild/cloudbuild.yaml

```diff
@@ -104,6 +105,7 @@ steps:
   env: [
     'BAZEL_REMOTE_CACHE=https://storage.googleapis.com/${_CACHE_BUCKET}/bazel-cache/${_DISTRO}-${_BUILD_NAME}',
     'LIBRARIES=${_LIBRARIES}',
```

Comment:
comment: `LIBRARIES` and `SHARD` seem to be serving the same purpose. One is for publish-docs, the other is for clang-tidy.

Having two variables seems non-ideal... but I am not the one doing the work to make the repo a nicer place to live :shrug:

Reply:
Yes.  The sharding for the documents is very ad-hoc.  I am going to leave it alone.